content/blogs/2022-11-19-create-data-pipeline-bigquery-google-cloud.md

```diff
@@ -251,31 +244,20 @@ triggers:
 
-For creating a sequential execution of tasks that has the dependency on the previous task, we need to provide all the sub-tasks in the parent task of the [**Sequential**](../plugins/core/tasks/flows/io.kestra.core.tasks.flows.Sequential.md) type. This sequential flow will be used to calculate the remaining stock at the product level using a Query plugin.
+## Data Lineage with Kestra ##
+With Kestra, the process of analyzing your flow (DAG) executions becomes intuitive and informative. Accessible through the "Execution" section, each execution instance is logged with specific details, such as its trigger source (manual or automated), as well as a unique Execution ID. This meticulously detailed record of each flow allows for comprehensive data lineage views encompassing Gantt charts, logs, topology, and outputs.
 
-While to run the tasks in parallel where the output of the task is not dependent, we need to specify all these tasks in type as [**Parallel**](../plugins/core/tasks/flows/io.kestra.core.tasks.flows.Parallel.md). In the above example, aggregating sales and stock data at the product level are two independent tasks which can be run in parallel using this plugin type.
+A key aspect of data lineage visualization is the Gantt chart, which provides a clear overview of the time taken for each task's data processing within the flow. In parallel, the Topology feature enables the user to identify and debug the sequence of tasks along with their dependencies.
 
-Here we can also provide the output of one task to the input to another task using the <code v-pre>{{outputs.task_id.output_parameter}}</code>`.
+![Gantt](/blogs/2022-11-19-create-data-pipeline-bigquery-google-cloud/Gantt.png)
 
-### Conclusion
+Accessing specific information about the flow is simple, requiring only the unique execution ID. This enables efficient data lineage tracking of the flow. You can leverage Kestra's null variable to add any execution identifier to separate different executions in the flow. Furthermore, you can inject a null variable into SQL queries for enhanced data tracking.
 
-We saw that [Kestra](https://www.kestra.io/) offers a variety of plugins **for creating a complete ETL/ELT pipeline using the GCP BigQuery** service along with features for monitoring the pipeline executions. It can be also used to schedule the Flow, provide easy debugging, and maintain the external dependencies. This blog post tries to showcase a few uses-case of how Kestra can be used along with the sample snippets for using the Kestra hands-on for you to start exploring the [Kestra](https://github.com/kestra-io/kestra) as it can cover endless use-cases.
+## Next Steps ##
+Stay tuned for our upcoming posts as we delve deeper into GCP integrations, providing more granular, real-world use cases and a step-by-step guide to leverage these powerful tools.
```

Comment:
```suggestion
If you would like to see more posts diving into GCP integrations or integrations with other cloud providers, let us know. We'd love to hear your feedback and the use cases you would like us to cover next. 
```

better not to promise the next posts unless you've already written that and plan to schedule -- this phrasing seems more flexible and invites readers to be part of the process without committing to anything yet